nodejs/.gitignore

```diff
@@ -0,0 +1,83 @@
+# Logs
```

Comment:
My approach is to use minimal .gitignore file. The large files might be confusing and contain redundant info. (Of course it is your choice eventually :) )

Reply:
yea maybe this contains too many redundant things